src/modules/Remote.rb

```diff
@@ -179,7 +200,8 @@ def WriteXinetd
         s = Ops.get_string(m, "service", "")
         next deep_copy(m) if !(s == "vnc1" || s == "vnchttpd1")
         Ops.set(m, "changed", true)
-        Ops.set(m, "enabled", @allow_administration)
+        Ops.set(m, "enabled", @mode == :xinetd) if s == "vnc1"
+        Ops.set(m, "enabled", @mode != :disabled) if s == "vnchttpd1"
```

Comment:
Similar to ```Convert``` a few comments above. Pls use just ```m["enabled"] = ...``` It helps us to clean the code ;-)